front-end/src/ui-components/VoteProgress.tsx

```diff
@@ -78,17 +82,34 @@ export default styled(VoteProgress)`
 	.voteNumbers {
 		display: inline-block;
 		color: grey_primary;
-		font-size: s;
+		font-size: sm;
 		margin-bottom: -1rem;
 	}
 
+	.nay {
+		position: absolute;
+		right: 0;
+	}
+
 	#passingThreshold {
 		display: inline-block;
+		position: absolute;
+		white-space: nowrap;
 		color: grey_primary;
-		font-size: s;
+		font-size: sm;
 		margin-top: -1rem;
 	}
 
+	.threshold-left {
+		position: absolute;
```

Comment:
I don't see the point in having it aligned left, if the percentage is less than 50%. The fact that it follows the `hr`is pretty cool IMO.

Reply:
It still follows the hr, just left-aligned. It's simply to avoid overflow when on the left side:
<img width="242" alt="Screenshot 2020-04-15 at 10 34 10" src="https://user-images.githubusercontent.com/7072141/79316583-cb8f8a00-7f04-11ea-889d-1542181f8fb8.png">

Reply:
ah nice, ok, that's great, my brain-built css parser didn't understand that. Need to actually test live.

Reply:
You can also use 
REACT_APP_HASURA_GRAPHQL_URL="https://kusama.polkassembly.io/v1/graphql"
REACT_APP_WS_PROVIDER="wss://kusama-rpc.polkadot.io/"
in frontend 
To test live kusama data

Reply:
yeah this is perfect :tada: